src/hypermedia-constants.js

```diff
@@ -677,6 +677,13 @@ export const Actions = {
 		skipSubscribeToParentNotifications: 'skip-subscribe-to-parent-notifications',
 		unsubscribeToParentNotifications: 'unsubscribe-to-parent-notifications'
 	},
+	discussions: {
```

Comment:
Now in alphabetical order

Reply:
I don't think there's really an alphabetical order in this file... 😅